### ISSUE TITLE
Add API Documentation

### ISSUE THREAD

**Issue by gregcmartin** [pwnlandia/mhn#37]
Add API endpoint documentation and links from the GUI

**Comment by zerokewl88** [comment]
PUSH - how can we find some api documentation to extract data from the database ?

**Comment by johnnykv** [comment]
@zerokewl88 I am the original developer of the Mnemosyne module - not sure how much @threatstream modified it, but I am pretty sure that most of the API documentation at http://johnnykv.github.io/mnemosyne/WebAPI.html still applies.

**Comment by andrew-morris** [comment]
@johnnykv Thanks so much for your work on Mnemosyne. 

One difference I've observed with the @threatstream implementation of Mnemosyne is that the web API which listens on tcp/8181 seems to be disabled. Attempting to access with ```netcat``` or ```curl``` yields a ```connection reset``` error immediately upon connection. 

```
$ curl -v xxx.xxx.xxx:8181
* Rebuilt URL to: xxx.xxx.xxx:8181/
* Hostname was NOT found in DNS cache
*   Trying X.X.X.X...
* Connected to xxx.xxx.xxx (X.X.X.X) port 8181 (#0)
> GET / HTTP/1.1
> User-Agent: curl/7.37.1
> Host: xxx.xxx.xxx:8181
> Accept: */*
> 
* Recv failure: Connection reset by peer
* Closing connection 0
curl: (56) Recv failure: Connection reset by peer
```

**Comment by jt6211** [comment]
Yes that's right. MHN uses mnenosyne for ingest only. The mhn web app implements its own APIs tailored for MHN. The mnenosyne api can be enabled but we kept it off buy default.

**Comment by onerkcll** [comment]
> Yes that's right. MHN uses mnenosyne for ingest only. The mhn web app implements its own APIs tailored for MHN. The mnenosyne api can be enabled but we kept it off buy default.

How we can enable mnenosyne api ? And if ı do could that broke something ?
